src/redux/books/books.js

```diff
@@ -4,7 +4,24 @@ const BOOK_REMOVED = 'BOOK_REMOVED';
 export const doAddBook = (book) => ({ type: BOOK_ADDED, playload: book });
 export const doRemoveBook = (id) => ({ type: BOOK_REMOVED, id });
 
-const initialState = [];
+const initialState = [
+  {
+    title: 'The Wolf and the Watchman',
+    author: 'Niklas Natt',
+    id: '1',
+  },
+  {
+    title: 'The Glided Wolves',
+    author: 'Oshani Chokshi',
+    id: '2',
+  },
+  {
+    title: 'The House of Broken Angels',
+    author: 'Luis Alberto',
+    id: '3',
+  },
+
+];
```

Comment:
- [ ] - [ ]  Your app runs with errors. I'm not able to see the home page.  After running the `npm install` then `npm start`.  Actually, I'm getting this error message 👇🏻 Please ensure that your app works as expected and runs without errors. 😉 👍🏻 

- Browser: 
![15SCR](https://user-images.githubusercontent.com/79058364/184839394-b6d5a6af-d0b4-438a-9371-c8c13696018f.png)

- Console:

![15scc](https://user-images.githubusercontent.com/79058364/184839658-0d1f93d6-a3cc-4198-aad1-93b33c5c31f1.png)

- [ ] I also noticed when I close the error message page, the book is not removed correctly, and sometimes clicking on a remove book button, removes all the books from that page. kindly fix this too 👍🏻